JGActionSheet/JGActionSheet.m

```diff
@@ -506,6 +506,10 @@ @interface JGActionSheet () <UIGestureRecognizerDelegate> {
     BOOL _anchoredAtPoint;
     CGPoint _anchorPoint;
     JGActionSheetArrowDirection _anchoredArrowDirection;
+    UIView *_anchorPointView;
+    UICollectionView *_anchorViewParentCollectionView;
```

Comment:
Have added readonly properties for anchorViewParentCollectionView and anchorViewTableCollectionView, and NSAssert if the view passed is not correct